examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -1716,7 +1715,7 @@ struct PhysicsServerCommandProcessorInternalData
 
 #ifdef STATIC_EGLRENDERER_PLUGIN
 	{
-		bool initPlugin = false;
```

Comment:
Not sure how to build plugin correctly (using the cmake build). So, I've replaced TinyRenderer with eglPlugin in the CMake and activated it here. To be resolved during final integration / build system cleanup shoud resolve it.

Reply:
calling 'loadPlugin' should init the plugin, no need to remove the initPlugin=false;